packages/platform-sdk-support/src/bignumber.ts

```diff
@@ -133,6 +133,17 @@ export class BigNumber {
 		return BigNumber.make(this.#value.multipliedBy(this.toBigNumber(value)));
 	}
 
+	/**
+	 * Returns the sum of the different values.
+	 *
+	 * @param {NumberLike[]} values
+	 * @returns {BigNumber}
+	 * @memberof BigNumber
+	 */
+	public static sum(values: NumberLike[]): BigNumber {
+		return values.reduce((carry: BigNumber, value) => carry.plus(value), BigNumber.ZERO);
```

Comment:
```suggestion
		return values.reduce((accumulator: BigNumber, currentValue) => carry.plus(value), BigNumber.ZERO);
```

Also add a type for `currentValue`.